### ISSUE TITLE
Cannot find module - 

### ISSUE THREAD

**Issue by mgifford** [GovTechSG/purple-a11y#380]
After pulling down the latest git repo, I can't seem to get past:  
  
 % npm run cli -- -c 2 -o a11y-scan-results.zip -u "http://localhost:8000" -w 360

  
 @govtechsg/purple-hats@0.10.5 cli  
 node dist/cli.js -c 2 -o a11y-scan-results.zip -u http://localhost:8000 -w 360

node:internal/modules/cjs/loader:1093  
 throw err;  
 ^

Error: Cannot find module '/Users/mgifford/Documents/GitHub/purple-a11y/dist/cli.js'  
   at Module.\_resolveFilename (node:internal/modules/cjs/loader:1090:15)  
   at Module.\_load (node:internal/modules/cjs/loader:934:27)  
   at Function.executeUserEntryPoint \[as runMain\] (node:internal/modules/run\_main:83:12)  
   at node:internal/main/run\_main\_module:23:47 {  
 code: 'MODULE\_NOT\_FOUND',  
 requireStack: \[\]  
}

Node.js v19.9.0  
  
 

I've tied to build it again from a fresh install but can't seem to get the results I need to get it running again from the command line.

**Comment by younglim** [comment]
Have you tried running `npm run build` before `npm run cli` ?
